streams/shared/src/main/scala/scalaz/stream/ZSink.scala

```diff
@@ -563,22 +654,37 @@ object ZSink {
       def extract(state: State) = IO.succeed(())
     }
 
-  def ignoreWhile[A](p: A => Boolean): ZSink[Any, Nothing, A, A, Unit] =
-    ignoreWhileM(a => IO.succeed(p(a)))
+  /**
+   * 🤔???
```

Comment:
Yeah you need @jdegoes for this one. I never grokked it entirely ;-)

Reply:
I think this should be called `dropWhile`. 

I reflexively create duals for the combinators I write... then try to figure out what they do and name them. 😄